### PR TITLE
Add support for custom contracts

### PR DESCRIPTION
**Pull request by adam-maj**

### CODE REVIEW

File: thirdweb/smart_contract.go [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+//
+// Example
+//
+// 	// The first parameter to the call function is the method name
+// 	// All other parameters to the call function get passed as arguments to your contract
+// 	balance, err := contract.Call("balanceOf", "{{wallet_address}}")
+//
+// 	// You can also make a transaction to your contract with the call method
+// 	tx, err := contract.Call("mintTo", "{{wallet_address}}", "ipfs://...")
+func (c *SmartContract) Call(method string, args ...interface{}) (interface{}, error) {
```
Comment (joaquim-verges) [inline thirdweb/smart_contract.go:117]: lol `interface{}` is just a fancy `any`

File: thirdweb/smart_contract.go [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+			if !ok {
+				return nil, fmt.Errorf(
+					"Argument %d should be of type string, but type %v was provided",
+					i+1,
+					reflect.TypeOf(arg),
+				)
+			}
+
+			arg = common.HexToAddress(parsedArg)
+		} else if strings.Contains(inputType, "int") {
```
Comment (joaquim-verges) [inline thirdweb/smart_contract.go:151]: do we handle bytes / bytes32 inputs?

File: cmd/thirdweb/custom_commands.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	Use:   "custom [command]",
+	Short: "Interact with an custom contract",
+	Args:  cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		log.Println("Please input a command to run")
+	},
+}
+
+var customGetCmd = &cobra.Command{
+	Use:   "set",
```
Comment (joaquim-verges) [inline cmd/thirdweb/custom_commands.go:25]: this name and description is wrong?

File: Makefile [hunk 4]
```diff
@@ -57,7 +57,7 @@ test-token-write:
 
 test-multiwrap-read:
 	./bin/thirdweb multiwrap getAll -a ${GO_MULTIWRAP} -k ${GO_PRIVATE_KEY} -u ${GO_ALCHEMY_RPC}
-	./bin/thirdweb multiwrap getContents -a ${GO_MULTIWRAP} -k ${GO_PRIVATE_KEY} -u ${GO_ALCHEMY_RPC}
+	# ./bin/thirdweb multiwrap getContents -a ${GO_MULTIWRAP} -k ${GO_PRIVATE_KEY} -u ${GO_ALCHEMY_RPC}
```
[the remark this answers is not in the document]
Comment (adam-maj) [reply Makefile:60]: Some are commented out for convenience so you can just run make test and all works straight

**Pull request merged**